workflow/cron/operator.go

```diff
@@ -169,40 +190,40 @@ func (woc *cronWfOperationCtx) terminateOutstandingWorkflows() error {
 }
 
 func (woc *cronWfOperationCtx) runOutstandingWorkflows() (bool, error) {
-	proceed, err := woc.shouldOutstandingWorkflowsBeRun()
+	proceed, missedExecutionTime, err := woc.shouldOutstandingWorkflowsBeRun()
```

Comment:
you have an invarient here: `proceed` === `!missedExecutionTime.IsZero()`

Reply:
Great point, I'll simplify.

Also someone's been doing too much TypeScript lately (`===`)